etc/kayobe/ipa.yml

```diff
@@ -27,6 +27,13 @@
 # List of additional Diskimage Builder (DIB) elements to use when building IPA
 # images.
 #ipa_build_dib_elements_extra:
+ipa_build_dib_elements_extra:
+  # Install dependencies required for the extra-hardware plugin.
+  - "ipa-extra-hardware"
+  # Add Mellanox element which installs packages and loads drivers required for
+  # Mellanox NICs.
+  - "mellanox"
+  - "stackhpc-ipa-hardware-managers"
```

Comment:
micronit: Adding something like  "#Add element to support verifying firmware versions" after the Mellanox element in case someone confuses stackhpc-ipa-hardware-managers to be associated with it?

Reply:
Will do.